yt/visualization/volume_rendering/tests/test_off_axis_SPH.py

```diff
@@ -0,0 +1,307 @@
+from yt.visualization.volume_rendering import \
+    off_axis_projection as OffAP
+from yt.testing import \
+    fake_sph_orientation_ds, \
+    assert_almost_equal, \
+    requires_module
+from yt.utilities.lib.pixelization_routines import \
+    pixelize_sph_kernel_projection
+import numpy as np
+from scipy.ndimage.filters import \
```

Comment:
Since scipy isn't a required dependency of yt you need to use the scipy on-demand import machinery for this. You also need to decorate the tests that use scipy with `@requires_module('scipy')`.

See here for the scipy on-demand imports: https://github.com/yt-project/yt/blob/master/yt/utilities/on_demand_imports.py#L151

It looks like you're going to need to add a wrapper for `ndimage`. Let me know if you have trouble with that.

See here for `requires_module`: https://github.com/yt-project/yt/blob/master/yt/testing.py#L598